### ISSUE TITLE
DBeaver can't connect to the database.

### ISSUE THREAD

**Issue by RIPMunakata** [dbeaver/dbeaver#22130]
### Description

Hello, I need to access a PostgreSQL database to make small changes to some records, and up until now, I had no issues with DBeaver. However, I had to reinstall it after formatting my Windows, and since then, I can no longer connect to the database.
When attempting to connect, the two errors in the screenshots below appear one after the other.
I have tried adding the JDBC Driver to the libraries section, but there has been no improvement.
Can someone help me? I have been trying to resolve this for hours and have not been successful so far.
![imagem_2023-12-07_164301213](https://github.com/dbeaver/dbeaver/assets/83376136/c5f0a83f-827f-4d81-b1a7-59d7eb6f6288)
![imagem_2023-12-07_164327074](https://github.com/dbeaver/dbeaver/assets/83376136/10f15b10-ff11-43ff-8693-5ea464f5f640)
![imagem_2023-12-07_164409308](https://github.com/dbeaver/dbeaver/assets/83376136/13ce02fb-a57d-4ac1-92e2-050525a602e2)

### DBeaver Version

23.3.0

### Operating System

Windows 10 Pro

**Comment by LonwoLonwo** [comment]
Hello @RIPMunakata 

Did you try to follow the advice from your first screen? About Preferences changing?

**Comment by RIPMunakata** [comment]
I tried now, and after disabling the "Use Windows trust store" option, I attempted to run the connection again. This time, a window appeared to download some drivers, I believe. I downloaded them and tried to establish the connection once again, but now a different error is appearing.
![imagem_2023-12-08_133434802](https://github.com/dbeaver/dbeaver/assets/83376136/2259c7df-ffe8-49e1-aade-c1515593bb0f)
